src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  // return days < 3 ? days * 40 : days < 7 ? days * 40 - 20 : days * 40 - 50;
+
+  const price = 40;
+  const longTermDiscount = 50;
+  const longTerm = 7;
+  const shortTermDiscount = 20;
+  const shortTerm = 3;
+  const totalCost = days * price;
+
+  if (days >= longTerm) {
+    return totalCost - longTermDiscount;
+  } else if (days >= shortTerm) {
+    return totalCost - shortTermDiscount;
+  }
```

Comment:
According to the checklist, avoid using `else` after `if` with `return` as the code after it won't be executed anyway. It simplifies the logic and makes the code cleaner. You can remove the `else` and just have a separate `if` statement for the short term condition.